python/ray/tests/test_autoscaler_fake_multinode.py

```diff
@@ -100,6 +101,51 @@ def i():
     # __example_end__
 
 
+@pytest.mark.parametrize("autoscaler_v2", [False], ids=["v1"])
+@pytest.mark.parametrize("placement_strategy", ["SPREAD", "STRICT_SPREAD"])
+def test_fake_autoscaler_placement_group_partial_idle(
+    autoscaler_v2, placement_strategy
+):
+
+    cluster = AutoscalingCluster(
+        head_resources={"CPU": 0},
+        worker_node_types={
+            "type-1": {
+                "resources": {"CPU": 2},
+                "node_config": {},
+                "min_workers": 0,
+                "max_workers": 10,
+            },
+        },
+        idle_timeout_minutes=0.1,
+        max_workers=10,
+        autoscaler_v2=autoscaler_v2,
+        provider={
+            "type": "fake_multinode",
+            "start_node_delay_s": 6,  # https://github.com/ray-project/ray/pull/51122
```

Comment:
Maybe use a bit more than 6 seconds to ensure it definitely exceeds 0.1 minute.

Reply:
👍